README.md

```diff
@@ -519,7 +519,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayush](https://github.com/ayushsgithub)
   - [Ayushi Vamne](https://github.com/Ayushi15-Vamne)
   - [Aziz Prabowo](https://github.com/azizp128)
-
+  - [Apurva Lagad](https://github.com/ApurvaLagad)
```

Comment:
Please add your name in line 423, after APT96.
Let us know if you need any help with that. 🙂